CODEOWNERS

```diff
@@ -1,2 +1,2 @@
 # See https://help.github.com/en/articles/about-code-owners
-* @alphagov/re-autom8 @alphagov/re-gsp
+* @alphagov/re-autom8 @alphagov/re-gsp !@issyl0
```

Comment:
I think you should have your entry on a seperate line

the format of this file is same as a gitignore I believe.

Reply:
actually I'm not sure ...lets just try it :-D